worldeditadditions/utils/parse/axes_parser.lua

```diff
@@ -0,0 +1,204 @@
+-- ██████   █████  ██████  ███████ ███████      █████  ██   ██ ███████ ███████
+-- ██   ██ ██   ██ ██   ██ ██      ██          ██   ██  ██ ██  ██      ██
+-- ██████  ███████ ██████  ███████ █████       ███████   ███   █████   ███████
+-- ██      ██   ██ ██   ██      ██ ██          ██   ██  ██ ██  ██           ██
+-- ██      ██   ██ ██   ██ ███████ ███████     ██   ██ ██   ██ ███████ ███████
+
+-- Error codes: https://kinsta.com/blog/http-status-codes/
+
+--- FOR TESTING ---
+local function unpack(tbl)
+	if table.unpack then
+		return table.unpack(tbl)
+	else return unpack(tbl) end
+end
+---------------
+
+local Vector3
+if worldeditadditions then
+	local wea = worldeditadditions
+	Vector3 = dofile(wea.modpath.."/utils/vector3.lua")
+else
+	Vector3 = require("worldeditadditions.utils.vector3")
+end
+
+local key_instance
+if worldeditadditions then
+	local wea = worldeditadditions
+	key_instance = dofile(wea.modpath.."/utils/parse/key_instance.lua")
+else
+	key_instance = require("worldeditadditions.key_instance")
+end
+
+--- Unified Axis Keywords banks
+local keywords = {}
+-- Direction keywords
+keywords.dir = {
+	["?"] = "front", f = "front",
+	facing = "front", front = "front",
+	b = "back", back = "back",
+	behind = "back", rear = "back",
+	l = "left", left = "left",
+	r = "right", right = "right",
+	u = "up", up = "up",
+	d = "down", down = "down",
+}
+-- Mirroring keywords
+keywords.mirroring = {
+	sym = true, symmetrical = true,
+	mirror = true, mir = true,
+	rev = true, reverse = true,
+	["true"] = true
+}
```

Comment:
This could be defined in a single declaration?
```suggestion
--- Axis keyword banks
local keywords = {
	-- Direction keywords
	dir = {
		["?"] = "front", f = "front",
		facing = "front", front = "front",
		b = "back", back = "back",
		behind = "back", rear = "back",
		l = "left", left = "left",
		r = "right", right = "right",
		u = "up", up = "up",
		d = "down", down = "down",
	}
	
	-- Mirroring keywords
	mirroring = {
		sym = true, symmetrical = true,
		mirror = true, mir = true,
		rev = true, reverse = true,
		["true"] = true
	}
}
```

Reply:
>This could be defined in a single declaration?

Yes. The separation was from a different attempt where functions were involved.